proto/poktroll/supplier/event.proto

```diff
@@ -34,4 +36,13 @@ message EventSupplierUnbondingEnd {
 // event will also be emitted immediately after this event.
 message EventSupplierUnbondingCanceled {
   poktroll.shared.Supplier supplier = 1 [(gogoproto.jsontag) = "supplier"];
+}
+
+// EventSupplierUnbondedBelowMinStake is emitted when an supplier has been unbonded
+// during settlement because their post-settlement stake dropped below the minimum
+// supplier stake requirement.
+message EventSupplierUnbondedBelowMinStake {
+  poktroll.shared.Supplier supplier = 1 [(gogoproto.jsontag) = "supplier"];
+  cosmos.base.v1beta1.Coin min_stake = 2 [(gogoproto.jsontag) = "min_stake"];
+  int64 unbonding_height = 3 [(gogoproto.jsontag) = "unbonding_height"];
 }
```

Comment:
Remove in favor of adding a `SupplierUnbondingReason` enum field on `EventSupplierUnbondingBegin` and `EventSupplierUnbondingEnd`.

Reply:
Restore in favor of adding a SupplierUnbondingReason enum field on EventSupplierUnbondingBegin and EventSupplierUnbondingEnd